From aa23783c39f96dd0b45f70d82557736a5b884c55 Mon Sep 17 00:00:00 2001
From: Cosimo Cecchi <cosimoc@gnome.org>
Date: Wed, 28 Jan 2015 12:43:58 +0000
Subject: filename-builder: use dash for time format separator

We can't use colons, but a separator here still looks nice. Use a dash.

diff --git a/src/screenshot-filename-builder.c b/src/screenshot-filename-builder.c
index 06706d6..5eade9c 100644
--- a/src/screenshot-filename-builder.c
+++ b/src/screenshot-filename-builder.c
@@ -130,7 +130,7 @@ build_path (AsyncExistenceJob *job)
       GDateTime *d;
 
       d = g_date_time_new_now_local ();
-      origin = g_date_time_format (d, "%Y-%m-%d %H:%M:%S");
+      origin = g_date_time_format (d, "%Y-%m-%d %H-%M-%S");
       g_date_time_unref (d);
     }
   else
-- 
cgit v0.10.1

